{
  try {
    C.throwException();
  }
 catch (  RuntimeException e) {
    return 1;
  }
  return 0;
}
